implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -60,16 +61,20 @@ private Converters() {}
     });
 
     @SuppressWarnings("unchecked")
-    static final Converter<Double> DOUBLE_CONVERTER = BuiltInConverter.of(2, (Converter & Serializable) value -> value != null ? Double.valueOf(value) : null);
+    static final Converter<Double> DOUBLE_CONVERTER = BuiltInConverter.of(2,
+            (Converter & Serializable) value -> value != null ? Double.valueOf(value) : null);
 
     @SuppressWarnings("unchecked")
-    static final Converter<Float> FLOAT_CONVERTER = BuiltInConverter.of(3, (Converter & Serializable) value -> value != null ? Float.valueOf(value) : null);
+    static final Converter<Float> FLOAT_CONVERTER = BuiltInConverter.of(3,
+            (Converter & Serializable) value -> value != null ? Float.valueOf(value) : null);
 
     @SuppressWarnings("unchecked")
-    static final Converter<Long> LONG_CONVERTER = BuiltInConverter.of(4, (Converter & Serializable) value -> value != null ? Long.valueOf(value) : null);
+    static final Converter<Long> LONG_CONVERTER = BuiltInConverter.of(4,
+            (Converter & Serializable) value -> value != null ? Long.valueOf(value) : null);
 
     @SuppressWarnings("unchecked")
-    static final Converter<Integer> INTEGER_CONVERTER = BuiltInConverter.of(5, (Converter & Serializable) value -> value != null ? Integer.valueOf(value) : null);
+    static final Converter<Integer> INTEGER_CONVERTER = BuiltInConverter.of(5,
```

Comment:
There's a lot of formatting adjustments in the PR to the implementation code.

I would not have expected any given the parent update I thought included all formatting updates.

Reply:
Do you want me to double check this on a fresh clone ?

Reply:
If you could that would be great.

Don't really know why there's so much reformatting happening

Reply:
Ok so a fresh clone :

```
[pkruger@pkruger tmp]git clone https://github.com/smallrye/smallrye-config.git
Cloning into 'smallrye-config'...
remote: Enumerating objects: 93, done.
remote: Counting objects: 100% (93/93), done.
remote: Compressing objects: 100% (62/62), done.
remote: Total 1563 (delta 22), reused 64 (delta 9), pack-reused 1470
Receiving objects: 100% (1563/1563), 268.46 KiB | 634.00 KiB/s, done.
Resolving deltas: 100% (540/540), done.
[pkruger@pkruger tmp]$ cd smallrye-config/
[pkruger@pkruger smallrye-config]$ mvn clean install
```
... build build build...

Then 

```
[pkruger@pkruger smallrye-config]$ git diff --stat
 config-sources/zookeeper/src/main/java/io/smallrye/configsource/ZooKeeperConfigSource.java                     | 18 +++++++++---------
 config-sources/zookeeper/src/test/java/io/smallrye/configsource/zookeeper/tests/TestSuite.java                 |  9 +++++----
 config-sources/zookeeper/src/test/java/io/smallrye/configsource/zookeeper/tests/ZooKeeperConfigSourceTest.java | 38 +++++++++++++++++++++++---------------
 implementation/src/main/java/io/smallrye/config/ConfigSourceMap.java                                           | 18 ++++++++++--------
 implementation/src/main/java/io/smallrye/config/Converters.java                                                | 92 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------
 implementation/src/main/java/io/smallrye/config/DirConfigSource.java                                           | 10 ++++++----
 implementation/src/main/java/io/smallrye/config/ImplicitConverters.java                                        | 17 +++++++++--------
 implementation/src/main/java/io/smallrye/config/PropertiesConfigSource.java                                    |  1 +
 implementation/src/main/java/io/smallrye/config/PropertiesConfigSourceProvider.java                            |  3 +--
 implementation/src/main/java/io/smallrye/config/SecuritySupport.java                                           |  3 +--
 implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java                                            |  8 ++++----
 implementation/src/main/java/io/smallrye/config/SmallRyeConfigBuilder.java                                     | 25 +++++++++++++++----------
 implementation/src/main/java/io/smallrye/config/SmallRyeConfigProviderResolver.java                            |  2 +-
 implementation/src/main/java/io/smallrye/config/inject/ConfigExtension.java                                    | 13 ++++++++-----
 implementation/src/main/java/io/smallrye/config/inject/ConfigInjectionBean.java                                |  9 ++++-----
 implementation/src/main/java/io/smallrye/config/inject/ConfigProducer.java                                     | 42 ++++++++++++++++++++++++++----------------
 implementation/src/main/java/io/smallrye/config/inject/ConfigProducerUtil.java                                 | 22 ++++++++++++----------
 implementation/src/test/java/io/smallrye/config/ConfigSourceMapTest.java                                       | 56 ++++++++++++++++++++++++++++++++++++--------------------
 implementation/src/test/java/io/smallrye/config/ConfigSourceWrapperTestCase.java                               | 11 ++++++++---
 implementation/src/test/java/io/smallrye/config/CustomConverterTestCase.java                                   | 14 +++++++++-----
 implementation/src/test/java/io/smallrye/config/EnvConfigSourceTestCase.java                                   | 38 +++++++++++++++++++-------------------
 implementation/src/test/java/io/smallrye/config/ImplicitConverterTestCase.java                                 |  1 -
 implementation/src/test/java/io/smallrye/config/KeyValuesConfigSource.java                                     |  5 ++---
 implementation/src/test/java/io/smallrye/config/MultiValueTestCase.java                                        |  7 ++-----
 implementation/src/test/java/io/smallrye/config/OptionalNumericalTypesTestCase.java                            |  5 ++---
 testsuite/extra/src/test/java/io/smallrye/config/test/collections/CollectionWithConfiguredValueTest.java       |  5 ++---
...
```

Can you maybe double check on your side. If this is true, maybe we get the formatting in first in a separate PR ?

Reply:
Sorry!

New parent must have been merged without formatting changes. I've created https://github.com/smallrye/smallrye-config/pull/132 to do the formatting

Reply:
No worries :)